### PR TITLE
feat: bring back custom qt style sheets for Windows

### PR DESCRIPTION
**Pull request by shenlebantongying**
close https://github.com/xiaoyifang/goldendict/issues/339

Some CSS files are slightly shorter than the original ones because some common values were set separately in each CSS file (the common ones were same in every file). When I removed the CSS files at https://github.com/xiaoyifang/goldendict/commit/587aad14716b0421c8bdcd1f31d61eed014ccd34 , those common values were merged into `qt-style.css`.

Other details aren't changed.

I moved the files to `/src/qtstyles/..` because we are unlikely to merge any code for those files from upstream now.

For the lingoes-blue, I added some extras that make it complete.

image

### CODE REVIEW

File: mainwindow.cc [hunk 1]
```diff
@@ -1172,20 +1172,30 @@ void MainWindow::applyQtStyleSheet( QString const & addonStyle, bool const & dar
...
 
 #if defined(Q_OS_WIN)
-  QFile winCssFile( ":/qt-style-win.css" );
+  QFile winCssFile( ":src/qtstyle/qt-style-win.css" );
   winCssFile.open( QFile::ReadOnly );
   css += winCssFile.readAll();
+
+  // Load an additional stylesheet
+  // Dark Mode doesn't work nice with custom qt style sheets,
+  if (!darkMode){
```
Comment (shenlebantongying) [inline mainwindow.cc:1192]: When dark mode is enabled, no additional style will be loaded.

**Comment by xiaoyifang** [comment]
image

**Comment by xiaoyifang** [comment]
lingoes.css etc is better rename  to qt-lingoes.css to distinguish with html css style.

**Comment by shenlebantongying** [comment]
This is the true lingoes blue, the side panels are supposed to be blue.
image

**Comment by shenlebantongying** [comment]
renamed to `qt-****.css` @xiaoyifang

**Comment by xiaoyifang** [comment]
the articleview's backgroud should also be blue.not yellow.
https://github.com/goldendict/goldendict/blob/e1e70e6a6c973da0f33bb76a8185f6a261709b20/article-style-st-lingoes-blue.css#L2

seems the background is not working .

**Comment by shenlebantongying** [comment]
The body only changes the lower layer and it is blue. The `.gdarticle`. is yellow

https://github.com/goldendict/goldendict/blob/e1e70e6a6c973da0f33bb76a8185f6a261709b20/article-style-st-lingoes-blue.css#L17-L18

image

**Pull request merged**